piwheels/master/cloud_gazer.py

```diff
@@ -71,15 +71,14 @@ def loop(self):
         for package, version, timestamp, action in self.pypi:
             if action == 'remove':
                 if version is None:
-                    self.logger.info(
-                        'disabled package %s (deleted)', package)
+                    self.logger.info('deleted package %s', package)
```

Comment:
Given this actually just marks the package for deletion rather than actually doing it, this should be `'deleting package %s'` (then we can have another info message in scribe when the deletion has actually occurred). Should make it a bit easier to understand the stuff in the logs.